internal/web3ext/web3ext.go

```diff
@@ -454,6 +454,8 @@ web3._extend({
 		new web3._extend.Method({
 			name: 'start',
 			call: 'validator_start',
+			params: 1,
+			inputFormatter: [null]
```

Comment:
does it need the input formatter?

Reply:
`inputFormatter: [null]` stands for single optional parameter